packages/emmett-esdb/src/testing/example.e2e.spec.ts

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable @typescript-eslint/no-floating-promises */
+import assert from 'node:assert';
+import { describe, it } from 'node:test';
+
+describe('Example e2e Test', () => {
+  it('should pass', () => {
+    assert.equal(1, 1);
+  });
+});
```

Comment:
Hi guys. Sorry for jumping in, isn't this a leftover?

Reply:
Yea, we can remove them, they were just examples of how to create tests using the native test runner